include/_pia.h

```diff
@@ -4,9 +4,14 @@
 /*                                                                           */
 /*                Internal include file, do not use directly                 */
 /*                                                                           */
+/* The Peripheral Interface Adapter (PIA) chip provides parallel I/O         */
+/* interfacing; it was used in Atari 400/800 and Commodore PET family of     */
+/* computers, for joystick and interrupts.                                   */
+/* - Sources; various + Wikpedia article on "Peripheral Interface Adapter"   */
```

Comment:
The `-` at the beginning of the line looks like an enumeration. In include/_antic.h you just put the "source" right behind the "text". I suggest to do the same here too.